tests/fast/BlobRestoreBasic.toml

```diff
@@ -7,7 +7,6 @@ disableTss = true
 storageEngineExcludeTypes = [4, 5]
 
 [[knobs]]
-bg_consistency_check_enabled = 0
```

Comment:
Also did you run 100k joshua tests (and maybe 50k of just *Blob* tests)?

Reply:
Yes, see updated description for test results. Thanks!